website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -718,3 +718,102 @@ Views with this configuration will be able to select from objects in `project_1.
 
 The `grant_access_to` config is not thread-safe when multiple views need to be authorized for the same dataset. The initial `dbt run` operation after a new `grant_access_to` config is added should therefore be executed in a single thread. Subsequent runs using the same configuration will not attempt to re-apply existing access grants, and can make use of multiple threads.
 
+<VersionBlock firstVersion="1.7">
+
+## Materialized views
+
+The BigQuery adapter supports [materialized views](https://cloud.google.com/bigquery/docs/materialized-views-intro)
+with the following configuration parameters:
+
+| Parameter                    | Type                 | Required | Default   | Change Monitoring Support | Reference                                        |
+|------------------------------|----------------------|----------|-----------|---------------------------|--------------------------------------------------|
+| `on_configuration_change`    | STRING               | NO       | `'apply'` | N/A                       |                                                  |
+| `cluster_by`                 | LIST[STRING]         | NO       |           | DROP/CREATE               | [Clustering](#clustering-clause)                 |
+| `partition_by`               | DICT                 | NO       |           | DROP/CREATE               | [Partitioning](#partition-clause)                |
+| `enable_refresh`             | BOOLEAN              | NO       | `True`    | ALTER                     | [Auto-refresh](#auto-refresh)                    |
+| `refresh_interval_minutes`   | FLOAT                | NO       | `30`      | ALTER                     | [Auto-refresh](#auto-refresh)                    |
+| `max_staleness` (in Preview) | INTERVAL             | NO       |           | ALTER                     | [Auto-refresh](#auto-refresh)                    |
+| `description`                | STRING               | NO       |           | ALTER                     |                                                  |
+| `labels`                     | DICT[STRING, STRING] | NO       |           | ALTER                     | [Labels](#specifying-labels)                     |
+| `hours_to_expiration`        | INTEGER              | NO       |           | ALTER                     | [Table expiration](controlling-table-expiration) |
+| `kms_key_name`               | STRING               | NO       |           | ALTER                     | [KMS encryption](#using-kms-encryption)          |
```

Comment:
The table can be one column thinner if doing a hyperlink like [`kms_key_name`](#using-kms-encryption) in the first column instead.

Reply:
Done